R/post_bug.R

```diff
@@ -23,9 +23,10 @@ components <- c("Accuracy", "Add-ons", "Analyses", "Documentation", "Graphics",
 #' @seealso To obtain and use the API key see create_bugzilla_key().
 #' [Webpage](https://bugs.r-project.org/bugzilla/enter_bug.cgi) for manual entry
 #' @return The ID of the issue posted.
-post_bug <- function(text, title, component, ...,
-                    version, product, host, key) {
+post_bug <- function(title, component, ...,
+                     version, product, host, key) {
     # Provide some checks/questions to the users
+    title <- readline(prompt = "Bug Title: ")
```

Comment:
Better to remove this and use the argument from the function. Otherwise we are asking twice the same information from the user.

Reply:
sure!